react.js

```diff
@@ -27,5 +27,6 @@ module.exports = {
     'react/destructuring-assignment': 'off',
     'react/require-default-props': 'off',
     '@studysync/persnickety/jsx-child-location': 'error',
+    'react/jsx-boolean-value': 2
```

Comment:
can you change this to `error` instead of `2`. 
I always forget which way the numbers are